Installing-and-Configuring/Prerequisites.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Prerequisites for Heads
+permalink: /Prerequisites
+nav_order: 1
+parent: Installing and configuring
+---
+
+Prerequisites for Heads
+===
+
+Required equipment
+---
+
+To install Heads on a physical device, you will need:
+
+* Supported motherboard or laptop ([see below](#supported-devices))
+* A heads compatible USB security dongle ([see below](#USB Security Dongles))
+* A heads compatible storage device for your public GPG key (USB flash drive)
+
+If your device requires external flashing ([see below](#supported-devices)),
+ you will also need:
+
+* [SPI Programmer](https://trmm.net/SPI_flash): ch341a programmer or raspberry
+ pi or bus pirate (ch341a is recommended for new users and can be found almost
+ [anywhere](https://www.amazon.com/s?k=ch341a+programmer).
+* Wires and a clip SOIC8 to connect your programmer of choice to the board’s
+ SPI flash chip(s).
+  * The [Pomona 5250](https://www.pomonaelectronics.com/products/test-clips/soic-clip-8-pin)
+   is suggested as it is high quality and easier to make contact with the pins.
+* A second computer to flash from (Try to use a recommended operating system:
+  Qubes or Debian 9 or Fedora 30)
+
+Supported devices
+---
+
+|Device| Board name|Firmware base|Requires external flashing| ME should be cleaned|Notes|
+|--|--|--|:--:|:--:|--|
+|Asus KGPE-D16|`kgpe-d16`|coreboot|X|||
+|Dell R630|`r630`|linuxboot||X||
+|Intel S2600wf|`s2600wf`|linuxboot||X||
+|Lenovo Thinkpad T420|`t420`|coreboot|X|X||
+|Lenovo Thinkpad T430|`t430-flash`|coreboot|X|X|initial flashed image|
```

Comment:
high maintenance. please point to boards directory.... This will definitely be unmaintaned

Reply:
WIP I'm pretty sure I just re-organized the original wiki content here.  

If I understand you would like to remove the section titled 'Supported devices'.  What about 'Emulated Devices'?  Should there be links to anywhere?

Reply:
I think, personally, that.devices supported should point to boards directory and refer to boards files for users to differenciate their usage. But it is not a strong objection. Just a maintainership contract.

Reply:
I added a link to https://github.com/osresearch/heads/tree/master/boards . The table includes useful information for comparing across devices which would be lost if it is removed.   How likely is it that the current information would change?  Is your concern the amount of work for adding NEW devices to keep the table current?